src/planner/plan/append_simple.cpp

```diff
@@ -39,7 +42,37 @@ void Planner::appendCreateTable(const BoundStatement& statement, LogicalPlan& pl
     auto info = createTable.getInfo();
     auto op = make_shared<LogicalCreateTable>(info->copy(),
         statement.getStatementResult()->getSingleColumnExpr());
-    plan.setLastOperator(std::move(op));
+
+    // If it is CREATE NODE TABLE AS, then copy as well
+    if (createTable.hasCopyInfo()) {
```

Comment:
discuss with @ray6080 on this comment.

I don't think putting a plan together at logic level is easy/elegant enough. I would do the following.

1. generate an inner logical plan for inner query
2. save the inner logical plan inside LogicalCreateTable
3. make sure we optimize the inner plan through optimizer (you can open an issue and leave this to me)
4. in mapper, create inner physical plan and physical create table. Put them together (with whatever strategy that is easier, dummySink or binary operator)

Reply:
@05st can you double check if the subquery plan goes through optimizer? you can open an issue for now if 

Let's discuss this when you're online. @andyfengHKU in my understanding, I think @05st should just reuse what `planCopyFrom`, and whatever we support with `planCopyFrom` should also be supported here in terms of optimizer. I don't feel "save the inner logical plan inside LogicalCreateTable" is easier, isn't that make the subplan invisible from plan tree? How would that help with optimizer?